src/components/DroughtHistory.vue

```diff
@@ -435,7 +445,18 @@ export default {
               trigger: `#${scrollIDFull}`,
               start: "top 25%",
               end: 'bottom 25%',
-              toggleClass: {targets: `#button-${scrollID}`, className:"currentButton"}, // adds class to target when triggered
+              toggleClass: {targets: `#button-${scrollID}`, className: "currentButton"}, // adds class to target when triggered
+              toggleActions: "restart reverse none reverse" 
+            },
+          })
+          // Show the map for each drought when in that drought period
+          tl.to(`#${scrollIDFull}`, {
+            scrollTrigger: {
+              markers: false,
+              trigger: `#${scrollIDFull}`,
+              start: "top 25%",
```

Comment:
I played around a little bit with where these triggers sit. I think 10% works a bit better because right now at 25%, the grey is present for quite a while after the map and button go back to "no drought" status. I imagine we'll tweak all these triggers a bit though once we're in mobile testing. 

![image](https://github.com/DOI-USGS/drought-timeline/assets/19958841/b78384e2-e6b3-45e2-aa9b-e838b0d5e942)